pyproject.toml

```diff
@@ -50,13 +50,22 @@ changelog = "https://github.com/semuconsulting/pyubx2/blob/master/RELEASE_NOTES.
 
 [project.optional-dependencies]
 test = [
```

Comment:
Just an observation, but is there no way this can be imported from requirements_build.txt rather than duplicating?

Reply:
AFAIK no easy way but will take under advisement